crates/holochain/src/conductor/cell.rs

```diff
@@ -232,13 +235,12 @@ impl Cell {
             PutAgentInfoSigned { .. }
             | GetAgentInfoSigned { .. }
             | QueryAgentInfoSigned { .. }
-            | QueryAgentInfoSignedNearBasis { .. } => {
-                // PutAgentInfoSigned needs to be handled at the conductor level where the p2p
-                // store lives.
-                unreachable!()
-            }
-            PutMetricDatum { .. } | QueryMetrics { .. } => {
-                // Same goes for metrics
+            | QueryGossipAgents { .. }
+            | FetchOpHashesForConstraints { .. }
```

Comment:
Can we just rename this `QueryOpHashes` to match the others?

Reply:
This query returns the agents and their arcs though?

Reply:
Yeah, rename fetch. "fetch X for constraints" just sounds like a long way to say "query X" to me.

Reply:
I renamed it, let me know if you object